src/helpers/testHelpers.tsx

```diff
@@ -20,3 +22,422 @@ export const render = (
+  order_book_details = "order book details mock response",
+  find_payment_paths = "find payment paths mock response",
+  find_strict_receive_payment_paths = "find strict receive payments paths mock response",
+  find_strict_send_payment_paths = "find strict send payments paths mock response",
+  all_payments = "all payments mock response",
+  payments_for_account = "payments for account mock response",
+  payments_for_ledger = "payments for ledger mock response",
+  payments_for_transaction = "payments for transaction mock response",
+  trade_aggregations = "trade aggregations mock response",
+  all_trades = "all trades mock response",
+  trades_for_account = "trades for account mock response",
+  trades_for_offer = "trades for offer mock response",
+  all_transactions = "all transactions mock response",
+  single_transaction = "single transaction mock response",
+  post_transaction = "post transaction mock response",
+  transactions_for_account = "transactions for account mock response",
+  transactions_for_ledger = "transactions for ledger mock response",
+}
+
+const server = setupServer(
+  // Friendbot
+  rest.get("https://friendbot.stellar.org", (_req, res, ctx) => {
+    return res(ctx.status(200));
+  }),
+  // Endpoints: accounts > accounts
+  rest.get("https://horizon-testnet.stellar.org/accounts", (_req, res, ctx) => {
+    return res(ctx.status(200), ctx.json({ test: ENDPOINT_RESPONSE.accounts }));
+  }),
+  // Endpoints: accounts > single account
+  rest.get(
+    "https://horizon-testnet.stellar.org/accounts/:accountId",
+    (_req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json({ test: ENDPOINT_RESPONSE.single_account }),
+      );
+    },
+  ),
+  // Endpoints: assets > all assets
+  rest.get("https://horizon-testnet.stellar.org/assets", (_req, res, ctx) => {
+    return res(
+      ctx.status(200),
+      ctx.json({ test: ENDPOINT_RESPONSE.all_assets }),
+    );
+  }),
+  // Endpoints: claimable balances > all claimable balances
+  rest.get(
+    "https://horizon-testnet.stellar.org/claimable_balances",
+    (_req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json({ test: ENDPOINT_RESPONSE.all_claimable_balances }),
+      );
+    },
+  ),
+  // Endpoints: claimable balances > single claimable balance
+  rest.get(
+    "https://horizon-testnet.stellar.org/claimable_balances/:claimableBalanceId",
+    (_req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json({ test: ENDPOINT_RESPONSE.single_claimable_balance }),
+      );
+    },
+  ),
+  // Endpoints: effects > all effects
+  rest.get("https://horizon-testnet.stellar.org/effects", (_req, res, ctx) => {
+    return res(
+      ctx.status(200),
+      ctx.json({ test: ENDPOINT_RESPONSE.all_effects }),
+    );
+  }),
+  // Endpoints: effects > effects for account
+  rest.get(
+    // TODO: fix URL to have only one / at the end
+    "https://horizon-testnet.stellar.org/accounts//effects",
```

Comment:
a few of these have a `//` - typo?

Reply:
Yeah, it looks like the bug is in the URL generator if there is a second param. I didn't want to fix this now in case it's used somewhere else and we can't test it.